src/geometries/EdgesGeometry.js

```diff
@@ -1,152 +1,185 @@
-import { BufferGeometry } from '../core/BufferGeometry.js';
-import { Float32BufferAttribute } from '../core/BufferAttribute.js';
-import * as MathUtils from '../math/MathUtils.js';
-import { Triangle } from '../math/Triangle.js';
-import { Vector3 } from '../math/Vector3.js';
-
+import { BufferGeometry } from "../core/BufferGeometry.js";
```

Comment:
Please retain the original formatting of the file. Otherwise it is hard to see what lines have actually changed.

Reply:
Sorry, I'll make the change immediately.